snapshots/devmapper/pool_device.go

```diff
@@ -306,6 +306,23 @@ func (p *PoolDevice) CreateSnapshotDevice(ctx context.Context, deviceName string
 		return metaErr
 	}
 
+	// The base device must be suspend before taking a snapshot to
+	// avoid corruption.
+	// https://github.com/torvalds/linux/blob/v5.7/Documentation/admin-guide/device-mapper/thin-provisioning.rst#internal-snapshots
```

Comment:
does it only work for new kernel?

Reply:
No. I just picked the latest non-rc version, because pointing `master` might break in future.

Reply:
I can use 5.4 (the latest LTS). 4.19 (the previous LTS) doesn't work well since the URL is different.

Reply:
Sorry for late reply. Merged